### DISCUSSION TITLE
Limit smooth scrolling to inside same buffer only

### DISCUSSION THREAD

**Discussion by bfulop** [neovide/neovide#1581] [Ideas]
Hello,

Thank you for this great project, Neovide is bringing the Neovim dream to reality!

I really love the smooth scrolling in Neovide, I think it really helps when navigating around code. It's much more than a gimmick to me, it gives me information about where I am in the file.

One improvement I would like though, is to restrict smooth scrolling only when jumping around in the same file. When switching between two buffers or using a Telescope picker, I find the scroll effect to be disorienting and superfluous. 

Is there a way currently to restrict the scope of smooth scrolling? I'm building Neovide locally, but have no knowledge in Rust.

Thanks!

**Comment by diocletiann** [comment]
I would like to also disable cursor animation when using the command line or whichkey if that is related.
